test/trustedform-url.test.js

```diff
@@ -1,5 +1,6 @@
 const {assert} = require('chai');
 const moment = require('moment');
```

Comment:
```suggestion
const {assert} = require('chai');
```
moment library is unused can be removed